projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -242,16 +250,21 @@ export default function SubscriptionControls( {
 						}
 					} }
 				/>
-				{ showSubscribersTotal && isPublicizeEnabled ? (
+				{ isPublicizeEnabled && (
```

Comment:
This will not work for published posts on simple sites with a free plan. See https://github.com/Automattic/jetpack/pull/28944/files#r1715184860

Reply:
Thanks, so basically needs updating to check if publicize module is enabled/disabled, or if Jetpack social is installed?

Reply:
Right.